bedrock/mozorg/templates/mozorg/mission.html

```diff
@@ -73,7 +73,7 @@ <h1 class="title-shadow-box">{{ _('We’re building a better Internet') }}</h1>
       <p>{{ _('Topics include support, products, and technologies') }}</p>
     </li>
     <li>
-      <h4><a href="/about/governance.html">{{ _('Governance') }}</a></h4>
+      <h4><a href="/about/governance/">{{ _('Governance') }}</a></h4>
```

Comment:
For the forums link you should be able to use `{{ url('mozorg.about.forums.forums') }}` too.